source/MapPanel.cpp

```diff
@@ -102,6 +105,22 @@ namespace {
 	const int HOVER_TIME = 60;
 	// Length in frames of the recentering animation.
 	const int RECENTER_TIME = 20;
+
+	bool HasMultipleLandablePlanets(const System &system)
+	{
+		const Planet *firstPlanet = nullptr;
+		for(auto &stellarObject : system.Objects())
```

Comment:
```suggestion
		for(auto &&stellarObject : system.Objects())
```
No need to make a copy in order to have a mutable reference.

Reply:
I don't see the copy here, both lines are in fact functionally identical. Am I missing something?